src/api/spec/jobs/report_to_scm_job_spec.rb

```diff
@@ -98,5 +110,88 @@
         subject
       end
     end
+
+    context 'when the event subscription payload contains filters' do
```

Comment:
You already have a unit test of `Workflows::Filter` and you know it behaves like it should behave. Why test all conditions again here?

Reply:
@hennevogel yes we wrote the test before we introduced `Workflows::Filter`. We removed now most of the tests from the job spec and only kept two to test that it successfully uses the Service Class to filter. Please have another look